crowdclick/settings/defaults.py

```diff
@@ -107,9 +110,23 @@
 LOGIN_REDIRECT_URL = '/'
 
 INFURA_ROPSTEN_ENDPOINT = ''
-MATIC_MUMBAI_ENDPOINT = 'https://rpc-mumbai.matic.today'
-CONTRACT_INSTANCE_ADDRESS = '0xA3d2C3F78652B93311363bBDEc3d16867F6D5C29'
-ACCOUNT_OWNER_PUBLIC_KEY = '0xDd2179e8D8755f810CdAe4a474F7c53F371FbB6A'
+web3_config_namedtuple = namedtuple('Web3_Config', ['endpoint', 'contract_address', 'public_key', 'private_key'])
+WEB3_CONFIG = {
+    'mumbai': web3_config_namedtuple(
+        endpoint='https://rpc-mumbai.matic.today',
+        contract_address='0xA3d2C3F78652B93311363bBDEc3d16867F6D5C29',
+        public_key='0xDd2179e8D8755f810CdAe4a474F7c53F371FbB6A',
+        private_key='a' * 64,
+    ),
+    'goerli': web3_config_namedtuple(
+        endpoint='https://goerli.net',
```

Comment:
@RiccardoBiosas I couldn't find the correct endpoint for goerli. Any help?

Reply:
there's no public endpoint, it must point to either one's own private node or to some node-as-a-service endpoint. i'll send you the infura project-id in private which will need to be declared on settings/default.py

Reply:
Ok :ok_hand: I'll keep a dummy value here. Production will be configured later.